tools/packaging/release/kata-deploy-binaries.sh

```diff
@@ -273,14 +275,14 @@ main() {
 	destdir="${workdir}/kata-static-${kata_version}-$(uname -m)"
 	info "DESTDIR ${destdir}"
 	mkdir -p "${destdir}"
-	install_image
 	install_kata_components
 	install_experimental_kernel
 	install_kernel
 	install_clh
 	install_qemu
 	install_qemu_virtiofsd
 	install_firecracker
+	install_image
```

Comment:
I see you changed the order here. Any particular reason?

Reply:
no, image took too long when testing the others.